explorer/helpers.go

```diff
@@ -0,0 +1,84 @@
+package explorer
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog/log"
+	"github.com/threefoldtech/zos/pkg/gridtypes"
+	"github.com/threefoldtech/zos/pkg/rmb"
+)
+
+// NewNodeClient Creates new node client from the twin id
+func NewNodeClient(nodeTwin uint32, bus rmb.Client) *NodeClient {
+	return &NodeClient{nodeTwin, bus}
+}
+
+func getNodeTwinID(nodeID string) uint32 {
+	queryString := fmt.Sprintf(`
+	{
+		nodes(limit:10, where:{nodeId_eq:%s}){
+		  twinId
+		}
+	}
+	`, nodeID)
+
+	result := []byte(query(queryString))
+
+	var res NodeResult
+	err := json.Unmarshal(result, &res)
+
+	if err != nil {
+		log.Error().Err(errors.Wrap(err, "couldn't parse json")).Msg("connection error")
+	}
+	nodeStats := res.Data.NodeResult
+	if len(nodeStats) > 0 {
+		return nodeStats[0].TwinID
+	}
+	return 0
+
+}
+
+// NodeStatistics Returns actual node Statistics from the node itself over the msgbus
+func (n *NodeClient) NodeStatistics(ctx context.Context) (total CapacityResult, err error) {
```

Comment:
This method (and the entire node client over rmb) is here https://github.com/threefoldtech/zos/blob/master-3/client/node.go#L129 
so why this has to be done again ?

Reply:
Because in the original method it returns total and used  separately, So it would take more effort to gather the results again

Reply:
What?

Reply:
I don't get what u trying to say. Also it's weird that this actually work. You code should instead be 
```
var result CapacityResult
// then do the n.bus.Call
```
Why do u have to define your type twice ?

Reply:
Got it and used the method from zos repo 👍🏻